### PR TITLE
Add twilio tap

### PR DESCRIPTION
**Pull request by tibortezsla**
## Problem

Introduce a new tap, `tap_twilio`. 

## Proposed changes

Add tap_twilio
- to the README
- .yml to pipelinewise-config
- to the docs
- .sample.yml

## Types of changes

What types of changes does your code introduce to PipelineWise?

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

- [x] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [x] Description above provides context of the change
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Unit tests for changes (not needed for documentation changes)
- [ ] CI checks pass with my changes
- [ ] Bumping version in `setup.py` is an individual PR and not mixed with feature or bugfix PRs
- [ ] Commit message/PR title starts with `[AP-NNNN]` (if applicable. AP-NNNN = JIRA ID)
- [ ] Branch name starts with `AP-NNN` (if applicable. AP-NNN = JIRA ID)
- [ ] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions

### CODE REVIEW

File: docs/installation_guide/installation.rst [hunk 1]
```diff
@@ -183,6 +183,8 @@ Here’s the list of the singer connectors and if they are installed by default
 +----------------------------+---------------------------------------------+----------------------------------+---------------------------------------+
 | tap-mixpanel               | ./install --connectors=tap-mixpanel         | YES                              |                                       |
 +----------------------------+---------------------------------------------+----------------------------------+---------------------------------------+
+| tap-twilio                 | ./install --connectors=tap-twilio           | YES                              |                                       |
```
Comment (tibortezsla) [inline docs/installation_guide/installation.rst:186]: How is it decided if it should be included in the default install?

Comment (Samira-El) [reply docs/installation_guide/installation.rst:186]: I think if we use it at TW then it's it's default install.

Comment (tibortezsla) [reply docs/installation_guide/installation.rst:186]: Then I guess YES should be good here, thanks!

File: dev-project/pipelinewise-config/tap_twilio.yml [hunk 2]
```diff
@@ -0,0 +1,93 @@
+---
+
+# ------------------------------------------------------------------------------
+# General Properties
+# ------------------------------------------------------------------------------
+id: "twilio"                           # Unique identifier of the tap
+name: "Twilio"                         # Name of the tap
+type: "tap-twilio"                     # !! THIS SHOULD NOT CHANGE !!
+owner: "somebody@foo.com"              # Data owner to contact
```
Comment (tibortezsla) [inline dev-project/pipelinewise-config/tap_twilio.yml:9]: @EricPulido what these values should be?

Comment (EricPulido) [reply dev-project/pipelinewise-config/tap_twilio.yml:9]: I am guessing the values should remain dummy variables for the PR and then when configuring in production we add the real values, right @koszti ?

Comment (koszti) [reply dev-project/pipelinewise-config/tap_twilio.yml:9]: yes, that's correct

**Review by koszti — DISMISSED** [review]

**Review by koszti — APPROVED** [review]

**Pull request merged**